_posts/2021-09-26-new-multiplayer-experience.md

```diff
@@ -0,0 +1,185 @@
+---
+title: New Multiplayer Experience
+author: TrueBrain
+---
+
+With 12.0 in feature-freeze, it is time to talk about what is in 12.0, and why we release early.
+
+In summary: 12.0 makes setting up multiplayer games painless.
+You no longer need to configure anything in your home network.
+Playing together are now just four simple steps:
+1) You just start a server.
+2) You set the server to invite-only or public.
+3) You share your invite-code with your friend.
+4) Your friend joins your server based on the invite-code.
+
+There is no need to setup port-forwarding, or anything like that.
+The above will just work, for everyone, as long as you have a working Internet connection.
+
+Depending on your connection, it causes us some extra infrastructure costs, but we hope to cover that with donations.
+If our regular donations (and thank you so much for those!) run out, we will let you know, and run some kind of fundraiser.
```

Comment:
Perhaps a "we are not currently in particular need of donations" of some kind?

Reply:
We do need donations, we are just not actively lobbying for it. So not sure what you mean?

Reply:
"we are not currently low on money"